_posts/2020-10-01-hacktoberfest.markdown

```diff
@@ -3,6 +3,28 @@ title: Hacktoberfest and Void Linux
 layout: post
 ---
 
+*Update:* Unfortunately due to recent changes that significantly
+increase the effort required from Void, we can no longer promise that
+your changes will count towards the 4 PRs required for Hacktoberfest.
+
+Many maintainers use `git am` in a script to apply patches and batch
+them into build units that the buildbot can work on at a time.  This
+precludes the application of labels unless the maintainer separately
+opens a browser, logs into GitHub, and interacts with the PR there.
+While the `git am` method should technically will work, we've noticed
+in the past that GitHub occasionally applies the wrong status to PRs
+using this method (ignores magic closing words) and we can't promise
+that PRs will be counted, as we aren't aware at this time why this
+sometimes works and sometimes doesn't.
+
+Its unfortunate that the rules were changed mid-run, as under the
+previous rules we could process contributions without additional
+effort, and we could accept many first time contributor's work to
```

Comment:
Consider changing `additional effort` to `disruptive alterations to our review process`.

Reply:
first time contributors' ?